packages/superset-ui-demo/storybook/shared/components/ResizableChartDemo.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState, ReactNode } from 'react';
+import ResizablePanel from './ResizablePanel';
+
+interface Size {
```

Comment:
minor. There is also a type `Dimension` in `@superset-ui/dimension`

```ts
import { Dimension } from '@superset-ui/dimension';
```

Reply:
Semantically, this is from `react-resizable`. I'll import it from there instead. [https://github.com/apache-superset/superset-ui/pull/557/commits/b297fea9fd59b4b742243e5c13f40eba872a58e7 ](https://github.com/apache-superset/superset-ui/pull/557/commits/b297fea9fd59b4b742243e5c13f40eba872a58e7)#diff-e296f83c808a6bf79336f4e585f97bd7

Reply:
sure. this is pretty much optional.